### DISCUSSION TITLE
Save settings per character

### DISCUSSION THREAD

**Discussion by morgs71** [smasherprog/EqTool#126] [Ideas]
Hey would it be possible to save settings based on character names?  For example on my rogue i was only to see my buffs and rely on the dps windows.  On my druid I need to see others buffs and don't use the dps window.  Would be cool if it would recognize which character we are on and adjust it accordingly.  Thank you !

**Comment by smasherprog** [comment]
Apologize about responding a year later, but this is all saved per character.
